pages/agent/v3/cli_oidc.md

```diff
@@ -1,8 +1,13 @@
 # buildkite-agent oidc
 
-The Buildkite Agent's `oidc` command allows you to request an OIDC token representing the current job. These tokens can be exchanged with federated systems like AWS.
+The Buildkite Agent's `oidc` command allows you to request an OpenID Connect (OIDC) token representing the current job. These tokens can be exchanged for specific roles on federated systems like AWS.
```

Comment:
```suggestion
The Buildkite Agent's `oidc` command allows you to request an OpenID Connect (OIDC) token representing the current job. These tokens can be exchanged for specific roles on federated systems like AWS, GCP, Azure and many others.
```

Tempted to expand the list of federated systems, even though we only have the AWS example in the other docs, just to let people's imagination run wild a bit.

Reply:
I'd agree :) Can also call out hash vault

Reply:
> Can also call out hash vault as a non-cloud provider example

hashicorp vault?

Reply:
... yes. My brain must have auto-corrected